platforms/serializers.py

```diff
@@ -0,0 +1,13 @@
+from rest_framework import serializers
+from .models import UserWord, Word
+
+
+class UserWordSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = UserWord
+        fields = ('__all__')
```

Comment:
never use the "lazy" definition for payload
describe a list of fields what you plan to use here